src/services/refactors/useDefaultImport.ts

```diff
@@ -0,0 +1,96 @@
+/* @internal */
+namespace ts.refactor.installTypesForPackage {
+    const actionName = "Convert to default import";
+
+    const useDefaultImport: Refactor = {
+        name: actionName,
+        description: actionName,

```

Comment:
this should be localizable, e.g. `getLocaleSpecificMessage(Diagnostics.Convert_to_default_import),`
name should be *not* be modified, it should act as an identifier

Reply:
If the locale is changed after boot this will have the wrong description, since this object is created immediately. This would affect other refactors too. Should we fix this?